### ISSUE TITLE
Quest drops not counting when looted

### ISSUE THREAD

**Issue by merloechoed** [echoes-emu/echoes-bt#751]
I've mentioned it on #749 but should probably be another issue worth of investigation, as it's related to quest drops but not the same problem.

As can be seen in screenshot, quest drops can be looted (4 sickle-fly wings in bag) but still not count towards the quest (which still displays 3/4 sickle-fly wings), causing the issue of leaving quest items left over after quest is turned-in.

![ScreenShot00066](https://user-images.githubusercontent.com/122577244/212373757-c37eb052-2531-415b-9c62-0c464722642e.jpg)

**Comment by merloechoed** [comment]
I can't say for sure if this only affects this specific quest, but I have a vague memory from open beta of this kind of problem happening in another quest, worth to investigate.

**Comment by bcp19** [comment]
Did the server happen to go down between the time you started the quest and this happening?

**Comment by merloechoed** [comment]
No, I don't think so, as as soon as I grabbed the quest, I went hunting the sickle-flies.

**Comment by echoes-emu** [comment]
This needs a repro, it's not happening for all collection quests all  the time

**Comment by woodx319** [comment]
Tried on neeker-friend wasn't able to repro. Wings ticked normally 1-2-3-4.
